dora/core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -2047,41 +2047,6 @@ public static String getSyncMetricName(long mountId) {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
-  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_ACTIVE_COUNT =
-      new Builder("Worker.CacheManagerThreadActiveCount")
-          .setDescription("The approximate number of block cache "
-              + "threads that are actively executing tasks in the cache manager thread pool")
-          .setMetricType(MetricType.GAUGE)
-          .setIsClusterAggregated(false)
-          .build();
-  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_CURRENT_COUNT =
-      new Builder("Worker.CacheManagerThreadCurrentCount")
-          .setDescription("The current number of cache threads in the cache manager thread pool")
-          .setMetricType(MetricType.GAUGE)
-          .setIsClusterAggregated(false)
-          .build();
-  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_QUEUE_WAITING_TASK_COUNT =
-      new Builder("Worker.CacheManagerThreadQueueWaitingTaskCount")
-          .setDescription("The current number of tasks waiting in the work queue "
-              + "in the cache manager thread pool, bounded by "
-              + PropertyKey.WORKER_NETWORK_ASYNC_CACHE_MANAGER_QUEUE_MAX)
-          .setMetricType(MetricType.GAUGE)
-          .setIsClusterAggregated(false)
-          .build();
-  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_MAX_COUNT =
-      new Builder("Worker.CacheManagerThreadMaxCount")
-          .setDescription("The maximum allowed number of block cache "
-              + "thread in the cache manager thread pool")
-          .setMetricType(MetricType.GAUGE)
-          .setIsClusterAggregated(false)
-          .build();
-  public static final MetricKey WORKER_CACHE_MANAGER_COMPLETED_TASK_COUNT =
-      new Builder("Worker.CacheManagerCompleteTaskCount")
-          .setDescription("The approximate total number of block cache tasks "
-              + "that have completed execution")
-          .setMetricType(MetricType.GAUGE)
-          .setIsClusterAggregated(false)
-          .build();
   public static final MetricKey WORKER_BLOCK_READER_THREAD_ACTIVE_COUNT =
```

Comment:
I only removed the obsolete metrics, instead of changing the existing ones. 

For example, the thread pool of this "Worker.BlockReaderThreadActiveCount" was actually renamed to `worker.grpc.reader.threads`. However, in order not to break existing monitors and dashboards, I chose not to rename them in this PR. I figure these all belong to another bigger batch of rework on the metrics?